src/main/java/com/epam/machine/repository/ClientRepositoryImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.machine.repository;
+
+import com.epam.machine.entity.Client;
+import lombok.Builder;
+import lombok.Data;
+
+import java.sql.*;
+
+@Data
+public class ClientRepositoryImpl implements ClientRepository{
+    final static String dataBaseUrl = "jdbc:postgresql://localhost:5432/Car";
```

Comment:
looks like you forgot private here and all constant should be in uppercase separated by `_`, so it will be like `DATA_BASE_URL`, apply it everywhere please